_layouts/map-item.html

```diff
@@ -11,52 +11,67 @@
     <script src="https://api.mapbox.com/mapbox-gl-js/v2.6.1/mapbox-gl.js"></script>
     <!-- Import Turf Module - Needed to calculate bounding boxes -->
     <script src='https://api.mapbox.com/mapbox.js/plugins/turf/v2.0.2/turf.min.js'></script>
+    <link rel="stylesheet" href="{{site.baseurl}}/assets/map-layout.css">
 </head>
 
 <body>
 
-    <div class="mapinfowrapper">
-        <span class="mapinfo">Scale {{ page.scale }}</span>
-        <span class="mapinfo">Years {{ page.years_published }}</span>
-        <span class="mapinfo">{{ page.nosheets }} map sheets</span>
-    </div>
-
-    <h1 class="page-heading">{{ page.title }}</h1>
+    <div style=" position: sticky; padding-top: 3%; padding-bottom: 2%; top: 0%; left: 3%; background-color: #fdfdfd; z-index: 0; min-width: 90%">
+        <div class="mapinfowrapper">
+            <span class="mapinfo">Scale {{ page.scale }}</span>
+            <span class="mapinfo">Years {{ page.years_published }}</span>
+            <span class="mapinfo">{{ page.nosheets }} map sheets</span>
+        </div>
+        <h1 class="page-heading">{{ page.title }}</h1>
 
-    <div id="buttonWrapper">
         <a href="{{ page.infourl }}" class="buttonlink" alt="Open Catalogue">Open Catalogue</a>
-        <button id="zoomto" class="btn-control" alt="Reset Map">Reset Map</button>
-        <a href="#openHelp" id="helpButton" class="buttonlink" alt="Help">&quest;</a>
     </div>
 
-    <div id="openHelp" class="overlay">
-        <div id="helpContent">
-            <a href="#" id="closeHelp">&times;</a>
-            <div id="helpText">
-                <h2>How to Use the OpenIndexMap-O-Matic</h2>
-                <div id="scrollHelp">
-                    <ul>
-                        <li>Click and drag on the map to move the tool.</li>
-                        <li>Zoom in and out by clicking the plus (+) or minus (-) buttons.</li>
-                        <li>Click on an outlined shape to see what maps are in the collection. Information about the maps in that area will be displayed below the map tool.</li>
-                        <li>Click the “Open Catalogue” button to navigate to the catalogue listing for the map series.</li>
-                        <li>Click the “Reset Map” button to return the tool to its default state.</li>
-                        <li>Click on each map listing to see more detail about a particular map.</li>
-                    </ul>
+
+    <div class="row" style="width: 100%; float: left; min-width: 90%;">

```

Comment:
It's good practice to keep CSS out of any html files and in one place, so that they are easier to find and change later. "id" can be used for one-use styling, and "class" can be reused across multiple elements. I'd move the styling to the map-layout.sccs file that you made.